src/components/tabController/apis/pageCarousel.api.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "TabController.PageCarousel",
+  "category": "layoutsAndTemplates",
+  "description": "TabController's PageCarousel component",
+  "note": "You must pass `asCarousel` flag to TabController and render your TabPages inside a PageCarousel",
+  "extends": ["ScrollView"],
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/TabControllerScreen/index.tsx",
+  "images": [],
+  "props": []
```

Comment:
Remove empty keys

Reply:
Removed only the props.. images kept same reason as previous comment